proposals/apple-mdm/base-setup-clix.md

```diff
@@ -0,0 +1,176 @@
+# MDM Base Setup
+
+Note: these can all be namespaced behind something like `apple-mdm` if needed. But I would like to avoid creating an mdm vertical vs. an osquery vertical to create an integrated product.
+
+## APNS
+### Step 1 - Generate CSR
+```
+$ fleetctl setup apns init
+Generating APNS keys at ./
+  fleet-apns-pki.crt ... done
+  fleet-apns-pki.key ... done
+  fleet-apns-push.key ... done
+  fleet-apns-push.csr ... done
+Send the .csr and .crt files to your friendly Fleet representative
+
+Keep the .key files. You will need them to complete this process. 
+
+Fleet will return a .p7m file to you within (TODO) business days. Use that file in the `fleetctl setup apns gen-req` command.
+```
+
+### Step 2 - Generate .req file from .p7m
+```
+$ fleetctl setup apns gen-req \
+--p7-file=~/path/to/*.p7 \
+--pki-key=~/path/to/fleet-apns-pki.key \
+--pki-crt=~/path/to/fleet-apns-pki.crt
+Working...
+Generated fleet-apns-push.req file at ./
+
+Visit identity.apple.com to upload this .req file. For more details, follow these instructions: NEED LINK HERE
```

Comment:
@lucasmrod Is there a link to an apple guide that tells users where to upload the .req file? 

Is it this? https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server/establishing_a_certificate-based_connection_to_apns

Reply:
Haven't found a good Apple link with instructions...

The instructions are documented in page 219 of [MDM-Protocol-Reference.pdf](https://developer.apple.com/business/documentation/MDM-Protocol-Reference.pdf):
![Screen Shot 2022-09-06 at 10 42 07](https://user-images.githubusercontent.com/2073526/188650572-4f16f404-f91e-4ce7-a0ed-a704ea51c19f.png)